src/test/java/hec/ensemble/ResSimTest.java

```diff
@@ -24,7 +24,7 @@ public void simulateResSim() throws Exception {
 
             TimeSeriesIdentifier tsid = new TimeSeriesIdentifier("Coyote.fake_forecast", "flow");
 
-            EnsembleTimeSeriesReader reader = db.getEnsembleTimeSeriesReader(tsid);
+            EnsembleTimeSeries reader = db.getEnsembleTimeSeries(tsid);
```

Comment:
the name "reader" is confusing given the change in the structure of the code, consider renaming that local variable.